src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs

```diff
@@ -899,14 +898,12 @@ private class InternalPartitionEnumerator : DynamicPartitionEnumerator_Abstract<
                     IEnumerator<TSource> sharedReader,
                     SharedLong sharedIndex,
                     SharedBool hasNoElementsLeft,
-                    object sharedLock,
```

Comment:
@weshaggard , yeah, good question but I double-checked before removing it.  This enumerator just wraps the enumerable, which uses the lock object, but the enumerator has no direct need for it itself.  The same code exists on the desktop (the field is also unused there).